bin/lib/zopen-install

```diff
@@ -120,7 +120,13 @@ installDependencies()
     echo "$dependencies" | xargs | tr ' ' '\n' | sort | while read dep; do
       cat <<ZZ >> "${name}/.depsenv"
 if [ -f "../${dep}/.env" ]; then
-  cd "../${dep}" && . ./.env; cd -
+  if [[ \$(type echo) == 'echo is a shell builtin' ]]; then
```

Comment:
Looks good.

Reply:
may be pushd > /dev/null and popd >/dev/null to prevent the print out.